packages/sources/oanda/schemas/env.json

```diff
@@ -0,0 +1,41 @@
+{
```

Comment:
We actually don't need the env.json file anymore in v3 since all of the settings are defined in custom configs

Reply:
Removed